ktor-http/ktor-http-cio/jvm/src/io/ktor/http/cio/websocket/DefaultWebSocketSession.kt

```diff
@@ -36,4 +36,12 @@ public actual interface DefaultWebSocketSession : WebSocketSession {
      * (for example due to connection failure).
      */
     public actual val closeReason: Deferred<CloseReason?>
+
+    /**
+     * Start WebSocket conversation.
+     *
+     * @param negotiatedExtensions specify negotiated extensions list to use in current session.
+     */
+    @OptIn(ExperimentalWebSocketExtensionApi::class)
```

Comment:
Why do we opt-in here? Shouldn't this function be experimental itself as it accepts a parameter of an experimental type?

Reply:
The other issue is that people may think that they need to invoke this start function as it's user's API. Can't we somehow hide it?

Reply:
`OptIn` Cause the start is regular API. We can mark it with @Internal with error